client/src/features/bulletins/Bulletins/Filters/DoubleSlider/DoubleSlider.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useRef, useEffect } from 'react';
+import './DoubleSlider.scss';
+import { CustomInput } from 'reactstrap';
+
+type Props = {
+  setMinPrice: (minPrice: number) => void;
+  setMaxPrice: (maxPrice: number) => void;
+};
+
+export const DoubleSlider = React.memo(({ setMinPrice, setMaxPrice }: Props) => {
```

Comment:
Questo dovrebbe essere un componente generico in `features/ui` e quindi anche da rendere tale